i8_terminal/service_result/service_result.py

```diff
@@ -34,8 +35,10 @@ def to_df(self, format: str = "default", style: str = "default") -> DataFrame:
     def to_json(self) -> Any:
         pass
 
-    def to_console(self) -> None:
-        pass
+    def to_console(self) -> Table:
+        df = self._df.copy()
+        df = self._format_df(df, format="humanize")
```

Comment:
Here also the parameter `format` should be a parameter of the `to_console` function with a default value of `humanize`